Sources/Fuzzilli/Execution/REPRL.swift

```diff
@@ -158,6 +164,9 @@ class REPRLExecution: Execution {
     private var cachedStderr: String? = nil
     private var cachedFuzzout: String? = nil
 
+    var differentialResult: Int
+    var differentialResultInputs: Int
```

Comment:
I guess I know what `differentialResult` is, but what is `differentialResultInputs`?

Reply:
`differentialResultInputs` is the number of inputs which contributed to the hash value (i.e., the number of calls zu fuzzilli_hash). I found this information useful during debugging but we might want to remove it for production